sections/faqs/browser-support.js

```diff
@@ -0,0 +1,11 @@
+import md from "components/md"
+
+const BrowserSupport = () => md`
+  ## Which browsers does styled-components support?
```

Comment:
Please go for the shorter:

> Which browsers are supported?

It's obvious what we're referring to so it's unnecessary to mention styled-components, especially in the FAQ :wink: